<ui version="4.0" >
 <comment>This program is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation; either version 2 of the License, or
(at your option) any later version.</comment>
 <class>Tabs_PreferencesUI</class>
 <widget class="QWidget" name="Tabs_PreferencesUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>TabBar_Config</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="lookGBox" >
     <property name="title" >
      <string>Look</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="placementLabel" >
          <property name="text" >
           <string>Placement:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="kcfg_TabPlacement" >
          <item>
           <property name="text" >
            <string>Top</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Bottom</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Left</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="spacer4" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>111</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_CloseButtons" >
        <property name="text" >
         <string>Show &amp;close button on tabs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_ShowTabBarCloseButton" >
        <property name="text" >
         <string>Show a close button on the &amp;right side of the tab bar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="focusGBox" >
     <property name="title" >
      <string>Focus</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="kcfg_BringToFront" >
        <property name="text" >
         <string>&amp;Focus new tabs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_FocusNewQueries" >
        <property name="text" >
         <string>Focus new &amp;queries</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mouseGBox" >
     <property name="title" >
      <string>Mouse</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="kcfg_MiddleClickClose" >
        <property name="text" >
         <string>&amp;Middle-click on a tab to close it</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer1" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>100</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
